components/protocol/component.go

```diff
@@ -111,10 +111,10 @@ func provide(c *dig.Container) error {
 				poa.NewProvider(validators),
 			),
 			protocol.WithNotarizationProvider(
-				slotnotarization.NewProvider(iotago.SlotIndex(ParamsProtocol.Notarization.MinSlotCommittableAge)),
+				slotnotarization.NewProvider(),
 			),
 			protocol.WithAttestationProvider(
-				slotattestation.NewProvider(iotago.SlotIndex(ParamsProtocol.Notarization.MinSlotCommittableAge)+slotattestation.DefaultAttestationCommitmentOffset),
+				slotattestation.NewProvider(slotattestation.DefaultAttestationCommitmentOffset),
```

Comment:
I had the same discussion with Andrea today, but we'll wait for Jonas for that, since he introduced it. I think it should be relatived to EvictionAge somehow